core/includes/theme.inc

```diff
@@ -2391,10 +2391,13 @@ function theme_mark($variables) {
  *     - children: A list of nested child items to render that behave
  *       identically to 'items', but any non-numeric string keys are treated as
  *       HTML attributes for the child list that wraps 'children'.
+ *     - type: The type of list to return for the nested/children items (e.g.
+ *       "ul", "ol").
```

Comment:
How about this:

`- type: the type of list to return for the children ('ul' or 'ol').`

Since you've already defined what "children" is in the preceding line, you can use that term. And since most people will single-quote 'ul' and 'ol' in usage, why not here? And 'ul' and 'ol' are the only choices, so no "e.g." needed.

Same comments would apply to the second (existing) usage of "type".

Reply:
```suggestion
 *     - type: The type of list to return for the children ('ul' or 'ol').
```